definitions/npm/react-bootstrap_v0.32.x/flow_v0.53.x-/react-bootstrap_v0.32.x.js

```diff
@@ -35,7 +35,7 @@ declare module "react-bootstrap" {
     componentClass?: ElementType,
     href?: string,
     type?: 'button' | 'reset' | 'submit',
-    bsStyle?: BsStyle,
+    bsStyle?: 'default' | 'primary' | 'success' | 'info' | 'warning' | 'danger' | 'link',
```

Comment:
Nice catch! Seems that react-bootstrap docs are not correct in this: https://react-bootstrap.github.io/components/buttons/ vs https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Button.js